tests/framework/installer/ceph_installer.go

```diff
@@ -798,6 +802,49 @@ func (h *CephInstaller) removeClusterFinalizers(namespace, clusterName string) {
 	logger.Infof("removed finalizers from cluster %s", objectMeta.Name)
 }
 
+func (h *CephInstaller) removeMonConfigMapFinalizers(namespace, configMapName string) {
```

Comment:
Why is this method not returning any error?

Reply:
[UninstallRookFromMultipleNS](https://github.com/rook/rook/blob/2983319b3994c337b2620f8d7728cfe304a0f79f/tests/framework/installer/ceph_installer.go#L572) doesn't have a return type.
So ig logging errors will work.